### PR TITLE
onboarding the rest of 1.25 release team

### PR DESCRIPTION
**Pull request by cici37**
onboarding the rest of 1.25 release team

### CODE REVIEW

**Comment by leonardpahlke** [comment]
> The following members of team release-team are not Kubernetes org members: krol3

ref: https://github.com/kubernetes/org/issues/3478

Oh, I see there seems to be a typo and @krol was invited, not @krol3 

cc @mrbobbytables

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -306,18 +306,18 @@ teams:
         - jeremyrickard # subproject owner / Technical Lead
         - jlbutler # 1.25 RT Lead Shadow
         - justaugustus # subproject owner / Chair
-        # - justin-chizer # 1.25 Bug Triage Shadow
+        - justin-chizer # 1.25 Bug Triage Shadow
         - jyotimahapatra # 1.25 CI Signal Shadow
         - katcosgrove # 1.25 Comms Lead
         - kcmartin # 1.25 Docs Lead
-        # - krol3 # 1.25 Docs Shadow
+        - krol3 # 1.25 Docs Shadow
```
Comment (reylejano) [inline config/kubernetes/sig-release/teams.yaml:313]:
```suggestion
        - krol # 1.25 Docs Shadow
```

Comment (leonardpahlke) [reply config/kubernetes/sig-release/teams.yaml:313]: krol3 should be correct but the org membership got send to krol :D. I can open a PR to fix it

**Pull request merged**